blender/arm/props_renderpath.py

```diff
@@ -280,6 +282,7 @@ class ArmRPListItem(bpy.types.PropertyGroup):
            description="A name for this item",
            default="Desktop")
 
+    rp_driver_list: CollectionProperty(type=bpy.types.PropertyGroup)
```

Comment:
What's the reason for moving `rp_driver_list` to here? It was a world property before and is now a renderpath property (due to moving it), but other code still expects this to be a world property. I think you need to move this back to its original place.

Reply:
I don't know, my fork got messy due to me working on the main branch by mistake. I have deleted in from github and forked it again twice in the last 6 months.
Now this pull request is an orphan branch rebased on main (which is untouched).